resources/views/changelog/show.blade.php

```diff
@@ -64,6 +64,11 @@ class="changelog__build-link"
                     @each('changelog._changelog_change', $changelogs, 'log')
                 </div>
 
+                @if(!Auth::check() || !Auth::user()->isSupporter())
```

Comment:
not sure if this should be only users without supporters, or also logged out ones, but that's how it is right now on the old website (i think)

Reply:
currently this is how the old site behaves.

Reply:
I'm not sure you even need the first check here?

Reply:
it will fail for logged out users otherwise.